scripts/llms_txt2ctx.py

```
@@ -0,0 +1,292 @@
+
+    Returns:
+        The content string or a placeholder message on failure or network block.
+    """
+    if url_or_path.startswith(('http://', 'https://')):
+        return f"<!-- Remote content skipped: {url_or_path} -->"
+
+    try:
+        repo_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        filepath = os.path.join(repo_root, url_or_path)
+        if os.path.exists(filepath):
+            with open(filepath, 'r', encoding='utf-8') as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)